haproxy/README.md

```diff
@@ -2,5 +2,12 @@
 This container will build HAProxy from source along with OpenSSL and Lua
 
 ## Execution
+If you're using podman, you need to ssh into your podman machine using
+```podman machine ssh``` 
+and navigate into this directory from there before running the next step. 
+
+If you've mounted the Users directory to your podman machine then you'll probably have to run something like
+```cd /Users/[your machine username]/[path to this directory]```
+
 Build the container which will compile, build and output the haproxy executable:
-`docker build -o . .`
\ No newline at end of file
+`docker build .`
```

Comment:
Does this still emit the output? the previous `-o` flag was important to get the haproxy binary: https://docs.docker.com/engine/reference/commandline/build/#output

Reply:
You are correct that it needs the ```-o``` flag, I went back to using that after realising I had to run it from inside the podman machine but forgot to update it